### PR TITLE
Scripts to automate sending messages in Telegram from command line

### PR DESCRIPTION
**Pull request by Namyalg**
This script can be used to automate the process of sending messages on Telegram from the command line

Issue relates to #94

### CODE REVIEW

File: System-Automation-Scripts/Automate-Telegram/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Automate Telegram ##
```
Comment (Neilblaze) [inline System-Automation-Scripts/Automate-Telegram/README.md]: Update to `# Automate Telegram`

File: System-Automation-Scripts/Automate-Telegram/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+- This script can be used to automate the process of sending messages in Telegram
+- Here the process of automation is achieved by using the framework Selenium.
+- Selenium is a portable framework for testing and automating web applications web applications
+
+## Working ##
```
Comment (Neilblaze) [inline System-Automation-Scripts/Automate-Telegram/README.md]: Update to `## Working`

**Review by Neilblaze — COMMENTED** [review]
Make the changes :)

**Comment by Namyalg** [comment]
Yes, I have made the changes

**Review by Neilblaze — APPROVED** [review]
Nice work :100:

**Comment by powerexploit** [comment]
> Yes, I have made the changes

NIce @Namyalg , can you add working image of script.

**Comment by Namyalg** [comment]
Okay, will do that

**Comment by Namyalg** [comment]
I have added all the requirements

**Review by Neilblaze — APPROVED** [review]
Good work! :100:

**Review by powerexploit — APPROVED** [review]

**Pull request merged**